playbooks/edx-east/edxapp.yml

```diff
@@ -29,4 +28,4 @@
    - role: newrelic
      when: COMMON_ENABLE_NEWRELIC
    - role: minos
-     when: COMMON_ENABLE_MINOS
+     when: COMMON_ENABLE_MINOS
```

Comment:
Niptuck: looks like we have a noise here. Let's avoid such changes when possible.

Reply:
removed.